Resources/Prototypes/Entities/Mobs/NPCs/animals.yml

```diff
@@ -0,0 +1,183 @@
+- type: entity
+  save: false
+  name: monkey
+  id: MonkeyMob_Content
+  description: A description.
```

Comment:
nice description

Reply:
I'm to lazy to write the descriptions for these, some ~~chump~~ new coder can do it